src/Symfony/Component/DomCrawler/Tests/Html5ParserCrawlerTest.php

```diff
@@ -60,6 +60,30 @@ public function testHtml5ParserNotSameAsNativeParserForSpecificHtml()
         $this->assertNotEquals($nativeCrawler->filterXPath('//h1')->text(), $html5Crawler->filterXPath('//h1')->text(), 'Native parser and Html5 parser must be different');
     }
 
+    public function testHasHtml5Parser()
+    {
+        $crawler = $this->createCrawler(null, null, null, true);
+
+        $reflection = new \ReflectionClass($crawler);
+        $property = $reflection->getProperty('html5Parser');
+        $property->setAccessible(true);
+        $html5ParserValue = $property->getValue($crawler);
+
+        $this->assertEquals(\Masterminds\HTML5::class, $html5ParserValue::class, 'Html5Parser must be a Masterminds\HTML5 instance');
```

Comment:
You probably want `assertInstanceOf` here.

Reply:
Thanks